services/binbase/values.yaml

```diff
@@ -15,6 +15,17 @@ fullnameOverride: ""
 podSecurityContext: {}
   # fsGroup: 2000
 
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
```

Comment:
А пустая строка равноценна _not set_ с точки зрения шаблонизатора?

Reply:
Да

Reply:
Используется  Go template
https://golang.org/pkg/text/template/#hdr-Actions

`{{if pipeline}} T1 {{end}}`
	If the value of the pipeline is empty, no output is generated;
	otherwise, T1 is executed. The empty values are false, 0, any
	nil pointer or interface value, and any array, slice, map, or
	string of length zero.
	Dot is unaffected.